File: CS/ViewModel/MainViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Collections.ObjectModel;
using DevExpress.Xpf.Docking;
using System.Windows.Media;

namespace WpfApplication
{
    public class MainViewModel 
    {

        public MainViewModel()
        {
            ChildViews.Add(new DocumentViewModel("Document1", "/Images/change.png"));
            ChildViews.Add(new DocumentViewModel("Document2", "/Images/create.png"));
            ChildViews.Add(new DocumentViewModel("Document2", "/Images/new-16x16.png"));
            ChildViews.Add(new PanelViewModel(Colors.Red));
            ChildViews.Add(new PanelViewModel(Colors.Blue));
            ChildViews.Add(new PanelViewModel(Colors.Green));
        }

        private ObservableCollection<object> _Children = new ObservableCollection<object>();


        public ObservableCollection<object> ChildViews
        {
            get { return _Children; }
            set
            {
                _Children = value;

            }
        }



        

    }
}

File: VB/ViewModel/MainViewModel.vb
Imports System.Collections.ObjectModel
Imports DevExpress.Xpf.Docking
Imports System.Windows.Media

Namespace WpfApplication

    Public Class MainViewModel

        Public Sub New()
            ChildViews.Add(New DocumentViewModel("Document1", "/Images/change.png"))
            ChildViews.Add(New DocumentViewModel("Document2", "/Images/create.png"))
            ChildViews.Add(New DocumentViewModel("Document2", "/Images/new-16x16.png"))
            ChildViews.Add(New PanelViewModel(Colors.Red))
            ChildViews.Add(New PanelViewModel(Colors.Blue))
            ChildViews.Add(New PanelViewModel(Colors.Green))
        End Sub

        Private _Children As ObservableCollection(Of Object) = New ObservableCollection(Of Object)()

        Public Property ChildViews As ObservableCollection(Of Object)
            Get
                Return _Children
            End Get

            Set(ByVal value As ObservableCollection(Of Object))
                _Children = value
            End Set
        End Property
    End Class
End Namespace
